styles/styles.css

```diff
@@ -1086,9 +1086,13 @@ main.blue-contract .section.section-with-title p {
   justify-content: center;
   letter-spacing: 1.25px;
   line-height: 100%;
-  padding: 17px 36px;
-  text-align: center;
+  max-width: 21.5em;
+  height: 48px;
+  padding: 0 36px;
+  text-align: left;
+  text-transform: uppercase;
```

Comment:
It was decided at the beginning of the project that the uppercase will be editor decision. Did this decision change? Now ALL buttons will have uppercase?